prusa/link/web/lib/wizard.py

```diff
@@ -34,9 +34,11 @@ def new_sn_format(serial):
     return NEW_SN_REGEX.match(serial)
 
 
-def sn_write_success():
+def sn_write_success() -> bool:
     """Check if the S/N was written successfully to the printer"""
-    polling: PrinterPolling = app.daemon.prusa_link.printer_polling
+    polling: PrinterPolling
+    # Yes, there can be none in there, nothing I can do about it, sorry mypy
+    polling = app.daemon.prusa_link.printer_polling  # type: ignore
```

Comment:
If there can be None here, we have to:
* set right, to can't be None
* check, if it happen, for example with assert, if it is only programmer issue.